### ISSUE TITLE
is it work both on android and ios ？

### ISSUE THREAD

**Issue by Yunnathai** [dokhacchung98/react-native-alpha-ijkplayer#1]
hi great work !
is it work both on android and ios ？

**Comment by congvn-nws** [comment]
Hi @jamninetyfive 
You should use [Easy Player](https://github.com/itgou/react-native-easy-ijkplayer), because my repository is fork from it

**Comment by dokhacchung98** [comment]
Hi @jamninetyfive
You should use [Easy Player](https://github.com/itgou/react-native-easy-ijkplayer), because my repository is fork from it

**Comment by Yunnathai** [comment]
> 

He seems not even maintaining...

**Comment by dokhacchung98** [comment]
Yeah, you can create issues for him. But i see this repository is enough the function.
